CSS/layout.css

```diff
@@ -0,0 +1,81 @@
+/*******  CSS/LAYOUT ********/
+
+body {
+  box-sizing: border-box;
+  width: 85%;
+  margin: 0 auto;
+  padding-top: 3em;
+}
+
+
+/*******  HEADER/ NAV  ********/
+
+.flex-header {
+    display:flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    justify-content: flex-start;
+    align-content: flex-start;
+    align-items: flex-start;
+    margin-bottom: 2em;
+}
+
+ .flex-header-1 {
+    width: 3.3em;
+}
+ .flex-header-2 {
+    flex-grow:2;
+    padding-left: .7em;
+}
+ .flex-header-3 {
```

Comment:
This is a place where a BEM-like class naming convention may be handy, for example if you name these:
```
.flex-header-logo {}
.flex-header-titles {}
.flex-header-nav-icons{}